### DISCUSSION TITLE
Kernel version for oraclelinux:8.6

### DISCUSSION THREAD

**Discussion by 50wu** [oracle/container-images#50] [Q&A] [answered]
Hi Team,

Oracle Linux 8 has two kinds of kernel:  Unbreakable Enterprise Kernel (UEK) and Red Hat Compatible Kernel (RHCK), which one the docker image oraclelinux:8.6 is using? From the kernel output info cannot figure out easily:

```
uname -a
Linux 505dbfe68b27 5.15.0-1017-gcp #23~20.04.2-Ubuntu SMP Wed Aug 17 02:46:40 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by totalamateurhour** [comment, marked as answer]
Remember that a docker image doesn't include a kernel. You are looking at the kernel on the host that is running the container engine.
